viam_cartographer.go

```diff
@@ -289,9 +289,13 @@ func New(
 	// do not initialize CartoFacade or Sensor Processes when using cloudslam
 	if svcConfig.UseCloudSlam != nil && *svcConfig.UseCloudSlam {
 		return &CartographerService{
-			Named:        c.ResourceName().AsNamed(),
-			useCloudSlam: true,
-			logger:       logger,
+			Named:          c.ResourceName().AsNamed(),
+			useCloudSlam:   true,
+			logger:         logger,
+			lidar:          timedLidar,
+			movementSensor: timedMovementSensor,
+			enableMapping:  optionalConfigParams.EnableMapping,
+			existingMap:    optionalConfigParams.ExistingMap,
```

Comment:
basically we were not passing anything into the cartographer service struct when cloudslam was enabled, so GetProperties was failing to actually get the properties of the slam service